src/sync/index.js

```diff
@@ -461,7 +461,7 @@ async function syncFinalResultSet(db, startBlock, endBlock, removeHexPrefix, top
 
             await db.Oracles.update(
               { topicAddress: topicResult.topicAddress },
-              { $set: { resultIdx: topicResult.resultIdx, status: 'WITHDRAW' } }, { multi: true },
+              { $set: { status: 'WITHDRAW' } }, { multi: true },
```

Comment:
can I confirm fe gets final result from `topic` model only?

Reply:
@frankobe Confirmed. The only time we use `oracle.resultIdx` is for the result history.